charts/linkerd-control-plane/values.yaml

```diff
@@ -379,10 +379,6 @@ policyValidator:
 # - The external tool needs to create the namespace with the label:
 #     config.linkerd.io/admission-webhooks: disabled
```

Comment:
It looks like this comment section being above `nodeSelector` is now messing with how it gets rendered in `README.md`. We don't want those changes as part of this.

@alpeb we're okay to remove this comment section as well though I think?

Reply:
Ah indeed, that needs to go